internal/cache/file/downloader/parallel_downloads_job.go

```diff
@@ -85,7 +85,17 @@ func (job *Job) parallelDownloadObjectToFile(cacheFile *os.File) (err error) {
 			rangeStart := start
 			rangeEnd := min(rangeStart+parallelReadRequestSize, end)
 
+			if goRoutineIdx == 0 {
+				if err = job.uberConcurrencySem.Acquire(downloadErrGroupCtx, 1); err != nil {
+					logger.Tracef("Error while acquiring semaphore resource: %v", err)
```

Comment:
we can avoid having this log statement as the err returned from this function will be logged in job.go

Reply:
Makes sense. Thanks!